src/app/shared/ipAddressHelper.js

```diff
@@ -10,6 +10,13 @@ module.exports = {
       } else {
         return unknown;
       }
+    } else if (req.headers && req.headers["x-forwarded-for"]) {
```

Comment:
I think, the existing code can be simplified for better readability an maintainability.  I recommended below codes for example, 
module.exports = {
  getIpAddress: (req) => {
    const headerForward = req.headers && (req.headers["forwarded"] || req.headers["x-forwarded-for"]);
    if (headerForward) {
      const ipAddress = headerForward.match(IPV4_MATCH);
      if (ipAddress) {
        return ipAddress[0];
      }
    }
    return unknown;
  },
};